### PR TITLE
feat: first draft for deleting patient

### LINKED ISSUE #618: Patientlist delete discharged patients
### Describe the feature

Patientlist delete discharged patients

### Why is this feature needed?

because there is no interaction with discharged patients currently and we need to have a delete feature

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by florianjs**
## Which issues does this pull request close?
closes #618

### CODE REVIEW

File: tasks/components/layout/PatientList.tsx [hunk 1]
```diff
@@ -221,14 +223,13 @@ export const PatientList = ({
...
+                          {
                             <Button color="negative" variant="textButton" onClick={event => {
                               event.stopPropagation()
-                              // TODO delete
+                              deletePatientMutation.mutate(patient.id)
                             }}>
                               {translation.delete}
                             </Button>
-                            */
                           }
```
Comment (DasProffi) [inline tasks/components/layout/PatientList.tsx:233]: remove the now unnecessary curly brackets

**Review by DasProffi — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -11383,3 +11379,7 @@ packages:
   /zod@3.22.2:
     resolution: {integrity: sha512-wvWkphh5WQsJbVk1tbx1l1Ly4yg+XecD+Mq280uBGt9wa5BKSWf4Mhp6GmrkPixhMxmabYY7RbzlwVP32pbGCg==}
     dev: false
+
+settings:
```
Comment (use-to) [inline pnpm-lock.yaml:11383]: Revert please. I want to see those settings at the beginning of this file.

File: tasks/mutations/patient_mutations.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ import {
   UnassignBedRequest,
   GetPatientDetailsResponse,
   GetPatientAssignmentByWardRequest,
-  GetPatientListRequest
+  GetPatientListRequest, DeletePatientRequest
```
Comment (use-to) [inline tasks/mutations/patient_mutations.ts:14]: Formatting please

**Review by use-to — DISMISSED** [review]
LGTM

**Review by DasProffi — APPROVED** [review]

File: tasks/components/layout/PatientList.tsx [hunk 4]
```diff
@@ -213,21 +215,23 @@ export const PatientList = ({
...
+                            deletePatientMutation.mutate(patient.id)
                           }}>
                             {translation.delete}
                           </Button>
-                          */
-                        }
-                      </div>
+                        </div>
+                      )}
+                    </Draggable>
```
Comment (DasProffi) [inline tasks/components/layout/PatientList.tsx:234]: Discharged patients aren't draggable and cannot be clicked, please remove this

**Review by DasProffi — CHANGES_REQUESTED** [review]

File: tasks/components/layout/PatientList.tsx [hunk 5]
```diff
@@ -213,21 +215,19 @@ export const PatientList = ({
...
-                          }}>
-                            {translation.delete}
-                          </Button>
-                          */
-                        }
-                      </div>
+                    <div
+                      key={patient.id}
+                      className={tw('flex flex-row pt-2 border-b-2 justify-between items-center')}
+                      onClick={() => updateContext({ wardId: context.wardId, patientId: patient.id })}
```
Comment (DasProffi) [inline tasks/components/layout/PatientList.tsx:221]: Remove the on lick discharged patients aren't selectable

**Review by DasProffi — CHANGES_REQUESTED** [review]

**Review by DasProffi — APPROVED** [review]

**Pull request merged**